### ISSUE TITLE
Options merger maybe upstream

### ISSUE THREAD

**Issue by Frizi** [mokkabonna/vue-vuelidate-jsonschema#5]
Hi!

I see you have added a pretty generic `validations` option merger and you are using it by invoking it directly here
https://github.com/mokkabonna/vue-vuelidate-jsonschema/blob/c0e192735192370c6d54a8e3cd5e00dcfac253a2/src/index.js#L166-L172

I like that idea and I simply forgot to prepare for such situation of using vuelidate with mixins. Would you consider moving that whole `merge-validation-options` module upstream?

When it's officially supported by Vuelidate, I think it should be possible to register it globally as
`Vue.config.optionMergeStrategies.validations`.

**Comment by mokkabonna** [comment]
Sure, I was planning to suggest that eventually actually, since I saw they weren't merged automatically. Are you going to copy it, or did you want me to do it? In the first case, let me know when it's present in vuelidate so I can use that directly.

Do you want to remove the dependency on lodash?

**Comment by Frizi** [comment]
Yes, I'd like to remove the lodash dependency. Actually looking at the existing merge strategies, it seems there is an existing one fitting this usecase. Could you try just plainly using `Vue.config.optionMergeStrategies. provide`? It uses the same strategy as `data` field, just without extra data specific error checking.

**Comment by mokkabonna** [comment]
It fails on overwriting and deleting validation options.
https://github.com/mokkabonna/vue-vuelidate-jsonschema/blob/master/test/specs/index.spec.js#L512-L593


So is a no go for me at least.
https://github.com/mokkabonna/vue-vuelidate-jsonschema#override-or-extend-validation
